### DISCUSSION TITLE
SEQ Syslog & ingress parsing

### DISCUSSION THREAD

**Discussion by b1tnoise** [datalust/seq-tickets#1714] [General]
Hi,

I'm new to the SEQ world.  At work, I'm used to work with splunk.  I work with 'table' outputs etc, but the searches and so on are based on parsing of the input.  In my SEQ at home I've set up the Syslog app and data is importing correctly, however there is no parsing, so it's all "flat" text.
![image](https://user-images.githubusercontent.com/59202103/207139297-58794961-091f-4bb1-877b-5581a178b4fd.png)
How can I make sure that, for example all the "IN" are mapped to a field named IN?
Thanks!
Danny

**Comment by nblumhardt** [comment]
Hi! Thanks for reaching out.

The syslog input itself doesn't support any additional parsing; the best bet for this kind of task is to use something like Fluent Bit to create an ingestion pipeline that receives syslog events, parses them, and then forwards them to Seq via its HTTP ingestion API.

There are a couple of moving parts involved, but it generally comes together very smoothly. Fluent Bit provides a syslog input:

https://docs.fluentbit.io/manual/pipeline/inputs/syslog

And you can plug this in in place of the tail input we use in our example here:

https://blog.datalust.co/tailing-a-log-file-or-folder-with-fluent-bit-and-seq/

The filter/output sections of the configuration should be pretty much identical.

Let me know if this helps!

Best regards,
Nick
